### DISCUSSION TITLE
Question: link_multiple types

### DISCUSSION THREAD

**Discussion by benlawraus** [anvilistas/anvil-extras#322] [Q&A] [answered]
Hi Thanks for anvil-extras.
How do you specify a link_multiple type in the link_tables schema?
The documentation has for link_single, but when I try for a field that is link_multiple, it is ignored.

**Comment by meatballs** [comment, marked as answer]
You'll probably need to share some code for us to be able to help here. There's not enough detail in your question.

However, there's no difference in what you specify for a multilink column. You provide the `linked_tables` dict and the module handles whether those are single or multiple linked columns.
